tests/test_integration.py

```diff
@@ -10,23 +10,24 @@
 too_long_chars = "x" * (2**20 + 1)
 
 cases = [
-    (get_provider("ollama"), os.getenv("OLLAMA_MODEL", OLLAMA_MODEL)),
-    (get_provider("openai"), "gpt-4o-mini"),
-    (get_provider("databricks"), "databricks-meta-llama-3-70b-instruct"),
-    (get_provider("bedrock"), "anthropic.claude-3-5-sonnet-20240620-v1:0"),
+    (get_provider("ollama"), os.getenv("OLLAMA_MODEL", OLLAMA_MODEL), dict(seed=3, temperature=0.1)),
```

Comment:
nit: comment on seed? temperature=0.1 makes sense
```suggestion
    # Set seed and temperature for more determinism, to avoid flakes
    (get_provider("ollama"), os.getenv("OLLAMA_MODEL", OLLAMA_MODEL), dict(seed=3, temperature=0.1)),
```